packages/components/src/components/TaskRunStatus/TaskRunStatus.test.js

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2019-2020 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import { renderWithRouter } from '../../utils/test';
+
+import TaskRunStatus from './TaskRunStatus';
+
+it('TaskRunStatus renders', () => {
+  const mockStore = configureStore();
+  const store = mockStore({ namespaces: { selected: 'default' } });
+
+  renderWithRouter(
+    <Provider store={store}>
+      <TaskRunStatus />
+    </Provider>
+  );
+});
+
+it('StepDetails renders terminated state', () => {
+  const mockStore = configureStore();
+  const store = mockStore({ namespaces: { selected: 'default' } });
+
+  renderWithRouter(
+    <Provider store={store}>
+      <TaskRunStatus status="terminated" />
+    </Provider>
+  );
+});
+
+it('StepDetails renders cancelled state', () => {
+  const mockStore = configureStore();
+  const store = mockStore({ namespaces: { selected: 'default' } });
+
+  renderWithRouter(
+    <Provider store={store}>
+      <TaskRunStatus status="False" taskRun={{ reason: 'TaskRunCancelled' }} />
+    </Provider>
+  );
+});
```

Comment:
'StepDetails' should be 'TaskRunStatus', and the tests updated to reflect the other changes